libs/api/domains/user-profile/src/lib/actorProfile.resolver.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  Args,
+  Mutation,
+  Parent,
+  Query,
+  ResolveField,
+  Resolver,
+} from '@nestjs/graphql'
+import { UseGuards } from '@nestjs/common'
+
+import {
+  CurrentUser,
+  IdsUserGuard,
+  ScopesGuard,
+  type User,
+} from '@island.is/auth-nest-tools'
+import { IdentityClientService } from '@island.is/clients/identity'
+
+import { UserProfileService } from './userProfile.service'
+import { ActorProfile, ActorProfileResponse } from './dto/actorProfile'
+import { UpdateActorProfileInput } from './dto/updateActorProfileInput'
+
+@UseGuards(IdsUserGuard, ScopesGuard)
```

Comment:
~ Redundant `ScopesGuard` as we are not doing any scope check here, and let the rest api do all the scope checking.